samples/sample-chat-app/build.gradle

```diff
@@ -46,4 +47,9 @@ dependencies {
     implementation platform('com.google.firebase:firebase-bom:28.1.0')
     implementation 'com.google.firebase:firebase-analytics'
     implementation 'com.google.firebase:firebase-messaging'
+    implementation "androidx.core:core-ktx:1.6.0"
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+}
+repositories {
+    mavenCentral()
```

Comment:
We have already added `mavenCentral()` to our list of repositories for all projects in the project root's `build.gradle` file.
```suggestion
```